src/main/java/mate/academy/model/MovieSession.java

```diff
@@ -0,0 +1,69 @@
+package mate.academy.model;
+
+import java.time.LocalDateTime;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.OneToOne;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "movie_sessions")
+public class MovieSession {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @OneToOne
+    @JoinColumn(name = "movie_id")
+    private Movie movie;
+    @OneToOne
```

Comment:
Can several movie sessions have the same cinema hall?

Reply:
ManyToOne here too.